Doxyfile

```diff
@@ -194,7 +194,7 @@ SEPARATE_MEMBER_PAGES  = NO
 # The TAB_SIZE tag can be used to set the number of spaces in a tab.
 # Doxygen uses this value to replace tabs by spaces in code fragments.
 
-TAB_SIZE               = 2
+TAB_SIZE               = 4
```

Comment:
I had to change this to get doxygen to format the code blocks properly.
https://www.stack.nl/~dimitri/doxygen/manual/markdown.html#mddox_code_blocks